src/pages/firefly-api/guides/concepts/terminology.md

```diff
@@ -49,30 +49,22 @@ For these APIs, the model will process an image using its context. For example,
 
 ## Reference Image
 
-A sample image provided to be used as a reference while generating image results (ie: such as the image parameter in the Generate Similar API, or the style  and structure.imageReference parameters in Generate Images).
+A sample image provided to be used as a reference while generating image results (ie: such as the image parameter in the Generate Similar API, or the style  and structure imageReference parameters in Generate Images).
```

Comment:
Provide a sample image as a reference to generate image results. For example, the `image` parameter in the Generate Similar API, or the style  and structure `imageReference` parameter in Generate Images.